core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -592,19 +591,19 @@ public void testToForgetPartitions() {
 
```

Comment:
Just curiously, L163 `Collections.unmodifiableList(new ArrayList<>());` can change to `List.of()`?